dataclients/kubernetes/testdata/routegroups/convert/failing-filter.log

```diff
@@ -1 +1 @@
-\[routegroup\] parse failed after token foo, last route id: , position 8: syntax error
+\[routegroup\] parse failed after token foo, last route id: , position 3: syntax error
```

Comment:
Note that position was wrong due to previous implementation using `"* -> "` as a prefix to parse filters.